datamap/mv/plan/src/main/scala/org/apache/carbondata/mv/plans/modular/ModularPlan.scala

```diff
@@ -96,6 +96,24 @@ abstract class ModularPlan
     _rewritten
   }
 
+  private var _rolledup: Boolean = false
```

Comment:
this coding style is different, please make it the same as in other place. change to `rolledup`

Reply:
changed